traitcuration/traits/templates/traits/trait_detail.html

```diff
@@ -236,11 +236,63 @@ <h4>New Term Information</h4>
       </div>
     </section>
     {% endif %}
+
+    <section class="section">
+      <h4>TRAIT HISTORY</h4>
+
+      {% if not user.is_anonymous %}
+      <div class="post-comment__container">
+        <div class="post-comment__header">
+          <span uk-icon="user"></span>
+          <span>Comment as {{user.get_full_name}}</span>
+          <button class="uk-button button button--primary post-comment__submit" onclick="commentButtonClicked()">Submit</button>
+        </div>
+
+        <div class="post-comment__body">
+          <textarea 
+          id="commentBody" 
+          class="uk-textarea post-comment__textarea" 
+          placeholder="Leave a commment..."></textarea>
+        </div>
+      </div>
+      {% endif%}
+
+      {% for event in history_events|dictsortreversed:"date" %}
+      {% if event.type == 'comment' %}
+      <div class="event event--comment">
+        <span class="event__title">{{event.content.author.get_full_name}} commented</span>
+        <span date="{{event.date.isoformat}}" class="date event__date"></span>
+        <p>{{event.content.body}}</p>
+      </div>
```

Comment:
When proposing new terms, they currently display like this:
![image](https://user-images.githubusercontent.com/10669118/89111461-f2d77c00-d44d-11ea-8e49-ba10f89997b8.png)

It probably shouldn't display "None" explicitly

Reply:
Should I just display the label then?

Reply:
Yes, something like:

> **Kirill Tsukanov** proposed a mapping to a new term on **August 4th 2020, 8:59**
> 
> **New term label**

Without the link

Reply:
Currently it contains a broken link to "None", like that:
![image](https://user-images.githubusercontent.com/10669118/89268910-1a3a6e80-d631-11ea-8c95-26b96b2f9c6e.png)